src/bundle/Resources/views/content/tabs/details.html.twig

```diff
@@ -0,0 +1,89 @@
+<div class="ez-list-toolbar">
+    <h2 class="ez-list-toolbar-label">{{ 'locationview.details.title'|trans({}, 'locationview') }}</h2>
+</div>
+
+<table class="ez-table-data">
+    <thead>
+        <tr>
+            <th>{{ 'locationview.details.creator'|trans({}, 'locationview') }}</th>
+            <th>{{ 'locationview.details.lastcontributor'|trans({}, 'locationview') }}</th>
+            <th>{{ 'locationview.details.versions'|trans({}, 'locationview') }}</th>
```

Comment:
@bdunogier - we will have to incorporate the label change in content details table. 
'Versions' will now be 'Published version'